src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -493,6 +493,19 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: Dec2Hex)
             },
+            {
+                 BuiltinFunctionsCore.Distinct,
```

Comment:
> BuiltinFunctionsCore.Distinct, [](http://example.com/codeflow?start=16&length=31)

Pull from static field on Library.cs, rather than shared BuiltinFunctionsCore.